src/Nethermind/Nethermind.BeaconNode.Test/BlockProducerTest.cs

```diff
@@ -65,15 +65,15 @@ public async Task BasicNewBlock()
             newBlock.Slot.ShouldBe(targetSlot);
             newBlock.Body.RandaoReveal.ShouldBe(randaoReveal);
             
-            Hash32 expectedParentRoot = new Hash32(Bytes.FromHexString("0x3111350140726cc0501223143ae5c7baad7f5a06764fcc7d444a657016e7d616"));
+            Hash32 expectedParentRoot = new Hash32(Bytes.FromHexString("0x91b06cbcd6dc97b89dc8b95e0b01a497932683b182e6c722ddfa10cd005b2180"));
```

Comment:
we should add a Hash32 constructor that takes a hex string. We may even already have one.

Reply:
:thumbsup: